src/test/init/features/firestore.spec.ts

```diff
@@ -41,8 +47,6 @@ describe("firestore", () => {
     });
 
     it("should error when cloud resource location is not set", async () => {
-      checkApiStub.returns(true);
-
       const setup = { config: {}, projectId: "my-project-123" };
 
       expect(firestore.doSetup(setup, {})).to.eventually.be.rejectedWith(
```

Comment:
These lint warnings are actually legit. Please use `await` on the `eventually` statements to ensure they resolve.